.github/workflows/integration.yml

```diff
@@ -35,10 +35,6 @@ jobs:
           neovim: true
           version: nightly
 
-      - name: Install plenary
-        run: |
-          git clone --depth 1 https://github.com/nvim-lua/plenary.nvim ~/.local/share/nvim/site/pack/vendor/start/plenary.nvim
-          ln -s $(pwd) ~/.local/share/nvim/site/pack/vendor/start
-
       - name: Run tests
-        run: nvim --headless --noplugin -c 'packadd plenary.nvim' -c "PlenaryBustedDirectory tests/ { minimal_init = 'tests/minimal_init.lua' }"
+        run: make test_lua
```

Comment:
reuse logic from `Makefile` so it's not duplicated.

PS. Maybe you think I should remove `Makefile`